tests/dummy/app/serializers/manager.js

```diff
@@ -12,9 +12,9 @@ export default RESTSerializer.extend(EmbeddedRecordsMixin, {
       serialize: true,
       deserialize: 'records',
     },
-  },
+  };
 
   keyForAttribute() {
-    return this._super(...arguments);
-  },
-});
+    return super.keyForAttribute(...arguments);
+  }
+}
```

Comment:
Not sure why keyForAttribute is here when it just calls its super, but I've left it alone in case it's there for a reason (I'm not super familiar with the EmbeddedRecordsMixin)